lua/bufferline/buffer.lua

```diff
@@ -40,6 +40,7 @@ local function get_activity(number)
 end
 
 local function get_name(opts, number)
+  --- @type nil|string
```

Comment:
There are a couple of places LLS could not determine the type, so I had to annotate it.